config/database.yml

```diff
@@ -24,6 +24,7 @@ default: &default
 development:
   <<: *default
   database: askdarcel_development
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
See http://edgeguides.rubyonrails.org/configuring.html#connection-preference on what Rails does if you provide both a URL and other attributes for the database, but the short answer is that it merges both sets of information, favoring the URL if there are conflicts. It is also totally cool with an empty string for the URL, so this shouldn't affect people who aren't using the environment variable.